### DISCUSSION TITLE
What do you mean "you own 100% of your data"?

### DISCUSSION THREAD

**Discussion by zigomi** [plausible/analytics#2157] [Anything Else]
My data is stored.... on a remote server.

The one which I do not own, nor do I have access to it, nor do I know  where precisely it's located except the country. Nor do I even know wether or not the owners of Plausable secretely copy my data to their local computer, directly from the Db.They may not, but there's no way for me to prove otherwise.

Nor can I even **prove** that you don't sell it, even if you claim that you don't, other than to take your word for it.

In what sense than do you claim that  "you own 100% of your data"? I don't. You have more access to my data than I do.

---

"Your data is stored in EU!"

And? What's special about EU in terms of storing data on its territory?  The presense of GDPR and other bullshit which helps nobody but annoys everybody?

**Comment by metmarkosaric** [comment]
Thanks for the feedback @zigomi! 

We've explained the details on data ownership in [our data policy](https://plausible.io/data-policy). Plausible allows you several options so for instance if you don't trust us, feel free to self-host our code on a server in your basement. That's possible too.

We believe GDPR is great for the privacy of the web and Plausible wouldn't even have been founded if GDPR didn't come into place. You can read more about why EU is a good place for our servers here https://plausible.io/blog/google-analytics-illegal

**Reply by zigomi** [reply]
>We've explained the details on data ownership in [our data policy](https://plausible.io/data-policy). Plausible allows you several options so for instance if you don't trust us, feel free to self-host our code on a server in your basement. That's possible too.

It's not wether or not I trust you. It's that  there's no way for anyone *to verify* your claims, is there?

**Reply by zigomi** [reply]
>We believe GDPR is great for the privacy of the web and Plausible wouldn't even have been founded if GDPR didn't come into place

May be. As well as there're other things about GDPR that are nonsense.

**Reply by ukutaht** [reply]
> It's not wether or not I trust you. It's that there's no way for anyone to verify your claims, is there?

It's in our terms of service. At the moment the only path to verifying our claims is to make us prove in court that we indeed follow our own terms of service.

If you have productive ideas on how you _could_ verify these claims in a simpler way, please let me know. If you're just here to vent, please stop.

**Reply by zigomi** [reply]
An average Joe wouldn't go to the court against Plausible, because it wouldn't be worth it. Nor the time, nor the reward, nor the money, nor the effort. He may be on other side of the Earth.

Once again: can an oridinary person, wherever he is, however much money he has, at any given time *verify* on his own that what you claim is true? By himself. Yes or no?


>If you have productive ideas on how you could verify these claims in a simpler way, please let me know. If you're just here to vent, please stop.

Why? So that you won't have to answer uncovenient questions?

**Reply by lightson** [reply]
>  As well as there're other things about GDPR that are nonsense.

I'd tend to disagree with that heavily. GDPR is bad for no-one except parasitic companies that rely on profiling and selling private data.

The popups are annoying, sure, but there are zero downsides to GDPR for the general public.

**Reply by lightson** [reply]
> can an oridinary person, wherever he is, however much money he has, at any given time verify on his own that what you claim is true? By himself. Yes or no?

You could say that about almost any claim made by any company globally. For example I could tell you that the sky is blue, but it you tried to verify that at night you'd likely call me a liar.

**Reply by zigomi** [reply]
>You could say that about almost any claim made by any company globally. For example I could tell you that the sky is blue, but it you tried to verify that at night you'd likely call me a liar.

2 questions:

1) How does the 1st imply the 2nd one?

2) "well.... everyone is a little bit of a lier... There's no way to verify anything with 100% certanty, under all conditions. So why even borther? Lie here, lie there... What's the problem?". 

You've just diverted.


1)  (Oh, I think I've understood)

---
My initial question still remains, though.

**Reply by zigomi** [reply]
>The popups are annoying, sure, but there are zero downsides to GDPR for the general public.

A downside is that a web developer, who runs a website or SaaS, and who may has never even had an intention to garther user's data in the 1st place, or not heavily so, has become required to complify with this nonsense. Otherwise, he may get heavily penalised.

And users are now getting annoyed by those pop-ups which they'll close in 0.5 second and which they won't read anyway.

**Reply by lightson** [reply]
I think you underestimate how many people engage with cookie banners.

My point stands: for the end user GDPR is fantastic.

Privacy and data handling shouldn't ever have been an afterthought, not for any developer.
